app/src/main/webapp/partials/errors.html

```diff
@@ -26,43 +26,52 @@
                 </form>
             </div>
         </div>
-        <div id="grid" ng-grid="gridOptions"></div>
     </div>
-    <div id="details" ng-controller="ErrorDetailsCtrl" class="panel panel-default" style="padding: 20px;">
-        <tabset type="tabs">
-            <tab heading="Error">
-                <br/>
-                <span ng-if="message">
-                    <h4>Error Message</h4>
-                    <p>{{message.errorMessage}}</p>
-                    <h4>Error Details</h4>
-                    <p>{{message.errorDetails}}</p>
-                </span>
-            </tab>
-            <tab heading="Payload">
-                <br/>
-                <form ng-if="message">
-                    <button type="button" class="btn btn-primary" ng-click='resubmitMessage()'>Resubmit message</button><br/><br/>
-                    <textarea class="payload">
-                        {{message.payload}}
-                    </textarea>
-                </form>
-            </tab>
-            <tab heading="Headers">
-                <br/>
-                <table ng-if="message" class="table table-striped">
-                    <thead>
-                        <tr><th>Name</th><th>Type</th><th>Value</th></tr>
-                    </thead>
-                    <tbody>
-                        <tr ng-repeat="header in message.headers">
-                            <td>{{header.name}}</td>
-                            <td>{{header.type}}</td>
-                            <td>{{header.value}}</td>
-                        </tr>
-                    </tbody>
-                </table>
-            </tab>
-        </tabset>
+    <div class="col-xs-12" id="errorDisplayPane">
```

Comment:
If you wrap this in another div with a `"row"` class, and also make the above comment's modifications, the two row's will line up cleaner.